Source/WebCore/rendering/RenderTableSection.h

```diff
@@ -236,6 +236,8 @@ class RenderTableSection final : public RenderBox {
 
     // This map holds the collapsed border values for cells with collapsed borders.
     // It is held at RenderTableSection level to spare memory consumption by table cells.
+    // FIXME: We should be using a smart pointer for the RenderTableCell. Note that cells are not getting
```

Comment:
Not sure I understand the comment saying “cells are not getting cleaned consistently”. What does it mean for a cell to be “cleaned”?

Reply:
It means they are sometimes left in the map after the cell gets destroyed. We thus end up with stale pointers as keys in the map. @alanbaradlay  is looking into it.